packages/react-magma-icons/src/icons/Aliases/AlertIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { IconProps } from '../../IconProps';
+import { ExpandMoreIcon } from '../Navigation/ExpandMoreIcon';
+export const AlertIcon = (props: IconProps) => {
+  console.warn('AlertIcon has been deprecated');
```

Comment:
These all need to be wrapped in a check for the process env and only run in dev mode. This will prevent the warnings in production mode and in test output and will allow rollup to tree-shake this console statement out of production builds